opentracing/README.md

```diff
@@ -8,13 +8,8 @@ For background:
 
 Everyday consumers of this `opentracing` package really only need to worry
 about a couple of key abstractions: the `StartSpan` function, the `Span`
-interface, and golang's `context.Context` idiom. Here are code snippets
-demonstrating some important use cases.
-
-A note about `context.Context`: the opentracing API here encourages (but does
```

Comment:
I did less coupling to `context.Context` in this version. I think I prefer the results... it's still awkward to propagate both a golang `context.Context` *and* our own `Span`s/`TraceContext`s, but IMO this makes the `opentracing` API easier to use and understand.